madmom/features/tempo.py

```diff
@@ -512,6 +726,7 @@ def write_tempo(tempi, filename, mirex=False):
     # format as a numpy array
     out = np.array([t1, t2, strength], ndmin=2)
     # write to output
-    np.savetxt(filename, out, fmt='%.2f\t%.2f\t%.2f')
+    if filename is not None:
+        np.savetxt(filename, out, fmt='%.2f\t%.2f\t%.2f')
```

Comment:
Ah yeah, I thought for the **verbose** online mode it would be nice to print the current most likely tempi and their relative strength to the console - just like it's done in the result files (tempo_1, tempo_2, strength). Since the write_tempo method not only prints those values into a file but also returns them I ended up using this for displaying the current estimation in the console:

![screen shot 2017-06-06 at 07 41 23](https://cloud.githubusercontent.com/assets/1398398/26815268/98e2960c-4a8b-11e7-8d1c-55b0fb2919ce.png)

For writing it to the console no writing to a textfile is needed (that's where the line of code comes from). Should I change this somehow?